spectral_cube/spectral_cube.py

```diff
@@ -2028,6 +2033,77 @@ def subcube_from_regions(self, region_list, allow_empty=False):
         # that we can now crop out
         return masked_subcube.minimal_subcube(spatial_only=True)
 
+    def _velocity_freq_conversion(self, ranges, veltypes, restfreqs):
+
+        header = self.wcs.to_header()
+
+        # Obtaining rest frequency of the cube in GHz.
+        restfreq_cube = header['restfrq'] * u.Hz
+
+        CTYPE3 = header['CTYPE3']
+
+        if 'RAD' in CTYPE3:
```

Comment:
instead of these if statements, use this:
https://github.com/radio-astro-tools/spectral-cube/blob/master/spectral_cube/spectral_axis.py#L73

Reply:
Thanks!